lib/Controller/ChatController.php

```diff
@@ -247,14 +253,17 @@ public function receiveMessages($token, $lookIntoFuture, $limit = 100, $lastKnow
 				$displayName = $guestNames[$comment->getActorId()];
 			}
 
+			list($message, $messageParameters) = $this->richMessageHelper->getRichMessage($comment);
+
 			return [
 				'id' => $comment->getId(),
 				'token' => $token,
 				'actorType' => $comment->getActorType(),
 				'actorId' => $comment->getActorId(),
 				'actorDisplayName' => $displayName,
 				'timestamp' => $comment->getCreationDateTime()->getTimestamp(),
-				'message' => $comment->getMessage()
+				'message' => $message,
+				'messageParameters' => $messageParameters,
```

Comment:
Should adjust the `docs/api-v1.md`

Reply:
Do you prefer me to explicitly document all the possible parameters or just point to the _RichMessageHelper.php_ file?

Reply:
No, just that there is `messageParameters`
See rich subject of activity: https://github.com/nextcloud/activity/blob/master/docs/endpoint-v2.md#activity-element
Maybe you can also link to https://github.com/nextcloud/server/issues/1706

Reply:
Done; I shamelessly ripped it from [the Notifications app](https://github.com/nextcloud/notifications/commit/19cc41a9ceb4c68e5b3093c8bd6c04d6541a880a#diff-be2c9ef579bbe4a1f1b0009530d415faR121)